File: app/src/main/java/berkea/vaportext/SplashActivity.java
package berkea.vaportext;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import com.crashlytics.android.Crashlytics;
import com.crashlytics.android.ndk.CrashlyticsNdk;
import io.fabric.sdk.android.Fabric;

/**
 * Created by berkeatac on 27/12/2017.
 */

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Fabric.with(this, new Crashlytics(), new CrashlyticsNdk());

        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
        finish();


    }
}

File: app/src/main/java/berkea/vaportext/Activity/SplashActivity.kt
package berkea.vaportext.Activity

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.crashlytics.android.Crashlytics
import com.crashlytics.android.ndk.CrashlyticsNdk
import io.fabric.sdk.android.Fabric

/**
 * Created by berkeatac on 27/12/2017.
 */

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Fabric.with(this, Crashlytics(), CrashlyticsNdk())

        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()
    }
}